config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-x-on-y.yaml

```diff
@@ -79,7 +53,7 @@ periodics:
       - --timeout=300m
 
 - name: ci-kubernetes-e2e-kubeadm-gce-stable-on-master
-  interval: 6h
+  interval: 4h
```

Comment:
I don't think you need to update the frequency of this TBH.

Reply:
just tweaks. can revert if needed.